CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Features
 1. [#1928](https://github.com/influxdata/chronograf/pull/1928): Add prefix, suffix, scale, and other y-axis formatting
 1. [#1886](https://github.com/influxdata/chronograf/pull/1886): Fix limit of 100 alert rules on alert rules page
+1. [#1945](https://github.com/influxdata/chronograf/pull/1945): Add Presentation Mode to `present` on URL detection
```

Comment:
Nice, though I find this is a bit unclear to understand. I've tried drafting a few and also find it difficult to communicate how to use it. How about `Allow toggling of presentation mode via URL by adding "present" as a query param`?

Reply:
how about `Add "present" option to URL to enable presentation mode` ?

Reply:
sounds good to me, though maybe throw the phrase `query param` in there, like `Add "present" query param option to URL to enable presentation mode`

Reply:
If I'm gonna say that, I'll just omit the "option". 👍

Reply:
works for me